packages/relative-values/models/src/animals.ts

```diff
@@ -0,0 +1,617 @@
+value_fiapo = value_reference_top_animal_org * tt(0.01 to 2)
+fiapo_item = {
+  name: "Federation of Indian Animal Protection Organisations",
+  year: 2022, 
+  slug: "federation_of_indian_animal_protection_organisations",
+  value: value_fiapo -> add_uncertainty -> ss
+}
+
+//Fish Welfare Initiative (FWI) is one of few organizations to work exclusively on improving the welfare standards of farmed fishes. The majority of their work takes place in India, but they also work in China and the Philippines. They run the Alliance for Responsible Aquaculture (ARA), which sets standards . FWI also engages in corporate outreach to improve fish welfare across the supply chain and conducts field research that informs standard setting for welfare improvements.  Read Review
+// Already modelled
+
+//Material Innovation Initiative (MII) works to reduce farmed animal suffering by increasing the availability of animal-free products and strengthening the animal advocacy movement. They produce research on animal-free alternatives to existing materials used in the fashion, automotive, and homegoods industries, and they work with investors, corporations, and entrepreneurs to support the development of new products and production processes. They also host events, such as conferences, to create opportunities for the development of new networks in the next-gen materials industry. MII has been an ACE Standout Charity since November 2021. Read Review.
+value_mii = gfi_item.value * tt(0.005 to 0.5)
+mii_item = {
+  name: "Material Innovation Initiative",
+  year: 2022, 
+  slug: "material_innovation_initiative",
+  value: value_mii -> add_uncertainty -> ss
+}
+
+// Mercy For Animals (MFA) operates in the U.S., Brazil, Canada, Hong Kong, India, and Mexico. MFA’s work focuses on strengthening the animal advocacy movement and improving animal welfare standards, as well as decreasing the consumption of animal products and increasing the availability of animal-free products. They engage in a variety of farmed animal advocacy programs, often involving the distribution of footage from their undercover investigations of factory farms, which they primarily promote via media outreach and online campaigns. MFA also engage in corporate and institutional outreach, research, lobbying, and policy work. They recruit and train volunteers and support farmers in transitioning away from animal agriculture. MFA was selected as an ACE Top Charity several times between 2014 and 2017, and they have been a Standout Charity since November 2021. Read Review.
```

Comment:
Did this text come from somewhere? "Read Review" seems like it was a link.

Reply:
Yes, these are the reviews from ACE itself, from. I found it useful to have them as I was estimating these, but happy to remove them now.

Reply:
Added in next commit